spec/EnableExpressErrorHandler.spec.js

```diff
@@ -0,0 +1,69 @@
+const ParseServer = require("../src/index");
+const express = require('express');
+const rp = require('request-promise');
+
+describe('Enable express error handler', () => {
+  beforeEach((done) => {
```

Comment:
Not sure why you need this, as you create a new server separately. Please remove if not required

Reply:
Ok. Really not needed